### PR TITLE
Get Upload Status Documentation Fix

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

Fixing responses on get upload status documentation

## Why?

NAC-1553

## This PR has:

- [ ] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [X] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

File: docs/openapi/generated/bulk-api/openapi.md [hunk 1]
```diff
@@ -208,23 +208,75 @@ Status Code **200**
...
+    "detial": "not found"
+  }
+}
+```
+
+> 401 Response
+
+```json
+{
+  "statusCode": "404",
```
Comment (bgfoshay) [inline docs/openapi/generated/bulk-api/openapi.md:275]: Question: this looks like another 404 example?

File: docs/openapi/generated/bulk-api/openapi.yaml [hunk 2]
```diff
@@ -177,7 +177,7 @@ paths:
                       error_message:
                         type: string
                         description: Error details as to why an upload failed
-                        example: Exception of type 'CsvHelper.FieldValidationException' was thrown.
+                        example: Exception of type 'CsvHelper.FieldValidationException' was thrown. conner
```
Comment (bgfoshay) [inline docs/openapi/generated/bulk-api/openapi.yaml:180]: Question: is this a typo?

Comment (ccaprio97) [reply docs/openapi/generated/bulk-api/openapi.yaml:180]: ya that was a test. forgot to take out

File: etl/docs/openapi/getUploadById.yaml [hunk 3]
```diff
@@ -49,7 +49,7 @@ get:
                   error_message:
                     type: string
                     description: "Error details as to why an upload failed"
-                    example: "Exception of type 'CsvHelper.FieldValidationException' was thrown."
+                    example: "Exception of type 'CsvHelper.FieldValidationException' was thrown. conner"
```
Comment (bgfoshay) [inline etl/docs/openapi/getUploadById.yaml:52]: Question: typo?

File: docs/openapi/generated/bulk-api/openapi.yaml [hunk 4]
```diff
@@ -187,10 +187,72 @@ paths:
...
                         example: COMPLETE
+              examples:
+                Successul upload 200:
+                  description: 200 Response to successful upload
+                  value:
+                    data:
+                      id: 2
+                      upload_identifier: '0x8DA63770FEF1551'
+                      created_at: '2022-07-11T19:54:27.903811Z'
+                      publisher: id_eaadmin_dev
```
Comment (bgfoshay) [inline docs/openapi/generated/bulk-api/openapi.yaml:198]: Question: didn't we remove id & publisher from the response?

File: etl/docs/openapi/getUploadById.yaml [hunk 5]
```diff
@@ -58,8 +58,70 @@ get:
...
+                    example: "COMPLETE"
+          examples:
+            Successul upload 200:
+              description: "200 Response to successful upload"
+              value:
+                data:
+                   id: 2
+                   upload_identifier: "0x8DA63770FEF1551"
+                   created_at: "2022-07-11T19:54:27.903811Z"
+                   publisher: "id_eaadmin_dev"
```
Comment (bgfoshay) [inline etl/docs/openapi/getUploadById.yaml:70]: Question: didn't we remove id & publisher from the response?

File: docs/openapi/generated/bulk-api/openapi.yaml (hunk 4 above)
Comment (ccaprio97) [reply docs/openapi/generated/bulk-api/openapi.yaml:198]: yup, i didnt edit that so it was out of date from before

File: docs/openapi/generated/bulk-api/openapi.yaml [hunk 6]
```diff
@@ -187,10 +187,70 @@ paths:
...
+                      error_message: Exception of type 'CsvHelper.FieldValidationException' was thrown.
+                      completed_at: '2022-07-11T19:54:28.303961Z'
+                      status: FAILED
+                Not Found:
+                  description: 404 Not found exception
+                  value:
+                    errors:
+                      status: '404'
+                      title: NotFoundException
+                      detial: not found
```
Comment (bgfoshay) [inline docs/openapi/generated/bulk-api/openapi.yaml:215]: Nitpick: spelling

File: docs/openapi/generated/bulk-api/openapi.md [hunk 7]
```diff
@@ -208,23 +208,73 @@ Status Code **200**
...
+```
+
+> 404 Not found exception
+
+```json
+{
+  "errors": {
+    "status": "404",
+    "title": "NotFoundException",
+    "detial": "not found"
```
Comment (bgfoshay) [inline docs/openapi/generated/bulk-api/openapi.md:264]: Nitpick: spelling

File: docs/openapi/generated/bulk-api/openapi.md [hunk 8]
```diff
@@ -208,23 +208,73 @@ Status Code **200**
...
+|» message|string|false|none|The message explaining the reason for access denied|
+
+Status Code **404**
+
+|Name|Type|Required|Restrictions|Description|
+|---|---|---|---|---|
+|» data|object|false|none|The response payload representing upload data.|
+|»» statusCode|string|false|none|Status of the upload|
+|»» title|string|false|none|The title of the error|
+|»» detail|string|false|none|The detialed description of the error|
```
Comment (bgfoshay) [inline docs/openapi/generated/bulk-api/openapi.md:225]: Nitpick: spelling

**Review by bgfoshay — APPROVED** [review]
👍 Thanks for taking care of all the nitpicks!

**Pull request merged**